ui/src/routes/_layout/organizations/$orgId/index.tsx

```diff
@@ -159,6 +143,14 @@ const OrganizationComponent = () => {
     manualPagination: true,
   });
 
+  if (orgError || prodError) {
+    return orgError || prodError;
+  }
+
+  if (!organization || !products) {
+    return <LoadingIndicator />;
+  }
+
```

Comment:
Would checking for `isError` and `isPending` be enough to assume that the data is available, as described [here](https://tanstack.com/query/latest/docs/framework/react/guides/queries#query-basics)?